packages/auth/src/SocialLogins/SocialLogins.tsx

```diff
@@ -18,7 +18,13 @@ export type SocialLoginsWithCompoundComponents = FC<SocialLoginsProps> & { Googl
 const stateMapper = ({ socialLoginsState }: AuthState) => socialLoginsState;
 
 export const SocialLogins: SocialLoginsWithCompoundComponents = (props: SocialLoginsProps) => {
-  const { firstLoad, socialLoginsConfig, error } = useAuth(stateMapper);
+  const { firstLoad, socialLoginsConfig, error, loadSocialLoginsConfiguration } = useAuth(stateMapper);
+
+  useEffect(() => {
```

Comment:
we are loading the social logins in requestAutorized saga function, why you are loading the config also here?

Reply:
when the user fails to login with SSO, we let him click on a button "Back to login", when he clicks on the button, there is no request to get the social logins providers.
@davidantoon